Cargo.toml

```diff
@@ -1,24 +1,23 @@
 [package]
-name = "hmac-sha1"
-version = "0.1.3"
-authors = ["Philip Woolford <woolford.philip@gmail.com>"]
-keywords = ["hmac", "sha1"]
-description = "Minimal implementation of HMAC-SHA1 in Rust."
-license-file = "LICENSE"
-repository = "https://github.com/pantsman0/rust-hmac-sha1"
-
-include = [
-    "**/*.rs",
-    "LICENSE",
-    "Cargo.toml",
-    ]
+name = "hmac-sha"
+version = "0.2.1"
+edition = "2018"
+authors = ["Philip Woolford <woolford.philip@gmail.com>", "Leonardo Razovic <lrazovic@gmail.com>"]
+keywords = ["crypto", "sha", "hmac", "digest"]
+description = "Minimal implementation of HMAC-SHA-{1,2,3} in Rust."
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/lrazovic/rust-hmac-sha"
+include = ["**/*.rs", "LICENSE"]
+categories = ["cryptography", "no-std"]
 
 [lib]
-name = "hmacsha1"
+name = "hmacsha"
 path = "src/lib.rs"
 
 [dependencies]
-sha1 = "^0.2.0"
+sha-1 = "0.9"
+sha2 = "0.9"
+sha3 = "0.9"
```

Comment:
Can these be removed? The create is just for hmac-sha1 and they don't appear to be referenced.